beacon-chain/p2p/service.go

```diff
@@ -67,6 +67,7 @@ type Service struct {
 	cfg                   *Config
 	startupErr            error
 	dv5Listener           Listener
+	isPreGenesis          bool // Variable determining if the chain is pre-genesis.
```

Comment:
```suggestion
	isPreGenesis          bool
```

This comment doesn't add any new information since the variable name is self describing.